properties/color/brand.json

```
@@ -2,12 +2,12 @@
   "color": {
     "brand": {
       "primary": {
-        "base":    { "value": "{color.base.palatinate.900.value}" }
+        "base":    { "value": "{color.snow.light.name}" }
       },
 
       "secondary": {
-        "base":    { "value": "{color.base.sky.900.value}" }
+        "base":    { "value": "{color.orange.name}" }
```

this color doesn't exists, you should change to a temporary color like this:
```suggestion
        "base":    { "value": "{color.base.melon}" }
```